spec/unit/libraries/default_spec.rb

```diff
@@ -20,6 +20,9 @@
 require 'spec_helper'
```

Comment:
We may should rename the spec to `visibility_spec.rb`. Do we intent to add more specs for other reporters?